README.md

```diff
@@ -1,3 +1,5 @@
 # pr_rebase_test
 
 this is the initial change on the master branch
+
+this change on feature branch was done before rebasing
```

Comment:
Master is 1 commit ahead of this branch, and you have conflicts. Rebase and solve them.